ui-tests/src/test/java/org/openmrs/reference/page/HeaderPage.java

```diff
@@ -9,7 +9,7 @@
 
     static final String HOME_ICON = "logo";
     static final String HOME_LINK_TEXT = "Home";
-    static final By LOGOUT = By.className("logout");
+    static final By LOGOUT = By.cssSelector("#navbarSupportedContent > ul > li.nav-item.logout > a");
```

Comment:
Why have this deeply nested a selector? Wouldn't `li.logout > a` or even just `.logout > a` do the job?

Reply:
Sure thanks.